tox.ini

```diff
@@ -22,7 +22,7 @@ commands =
 deps =
     {[testenv]deps}
 commands =
-    python3 -m unittest tests/test_unit.py tests/test_helpers.py
+    python3 -m unittest tests/test_unit.py tests/test_helpers.py tests/test_markdown.py
```

Comment:
Is there a way to run all tests in the `tests` directory rather than having to list out each file? See https://docs.python.org/3/library/unittest.html#test-discovery. Perhaps:

```
python3 -m unittest discover tests
```

Reply:
I've left a note on https://github.com/googleapis/sphinx-docfx-yaml/issues/115 to look into this in a followup PR. There's a `test_yaml.py` in the test directory that's not making this easy to discover just set select examples.